src/elements/content-sidebar/activity-feed/task-new/Assignees.js

```diff
@@ -0,0 +1,113 @@
+// @flow strict
+import * as React from 'react';
+import { injectIntl, FormattedMessage } from 'react-intl';
+import type { InjectIntlProvidedProps } from 'react-intl';
+import { Flyout, Overlay } from '../../../../components/flyout';
+import Tooltip from '../../../../components/tooltip';
+import messages from '../../../common/messages';
+import AssigneeStatus from './AssigneeStatus';
+import { TASK_NEW_APPROVED, TASK_NEW_REJECTED, TASK_NEW_COMPLETED, TASK_NEW_NOT_STARTED } from '../../../../constants';
+
+const MAX_AVATARS = 3;
+
+type Props = {|
+    assignees: TaskAssigneeCollection,
+    getAvatarUrl: GetAvatarUrlCallback,
+    maxAvatars?: number,
+    ...InjectIntlProvidedProps,
+|};
+
+const statusMessages = {
+    [TASK_NEW_APPROVED]: <FormattedMessage {...messages.tasksFeedStatusAccepted} />,
+    [TASK_NEW_REJECTED]: <FormattedMessage {...messages.tasksFeedStatusRejected} />,
+    [TASK_NEW_COMPLETED]: <FormattedMessage {...messages.tasksFeedStatusCompleted} />,
+    [TASK_NEW_NOT_STARTED]: <FormattedMessage {...messages.tasksFeedStatusNotStarted} />,
+};
+
+const AssigneeTooltipLabel = React.memo(({ user, status }) => {
+    const statusMessage = statusMessages[status] || null;
+    return (
+        <div className="bcs-task-assignment-tooltip-text">
+            <div>
```

Comment:
Try to avoid non-semantic markup when possible. Should this be a heading tag?

Reply:
We are going to aria-label the avatar/tooltip into a single piece of information for screenreaders as a followup